mousestyles/dynamics/__init__.py

```diff
@@ -263,3 +263,74 @@ def get_prob_matrix_small_interval(string_list):
         if sum(Mat_prob[k, :]) != 0:
             Mat_prob[k, :] = Mat_prob[k, :]/sum(Mat_prob[k, :])
     return Mat_prob
+
+
+def mcmc_simulation(mat_list, n_per_int):
```

Comment:
Should we have a naming style for the function to be verb_none? Maybe a better name would be simulate_mcmc? But just a style thing